deploy/crds/db_v1alpha1_cassandracluster_crd.yaml

```diff
@@ -10,4 +10,313 @@ spec:
+              properties:
+                dc:
+                  description: List of DC defined in the CassandraCluster
+                  items:
+                    properties:
+                      labels:
+                        additionalProperties:
+                          type: string
+                        description: Labels used to target Kubernetes nodes
+                        type: object
+                      name:
+                        description: Name of the CassandraDC
+                        type: string
+                      nodesPerRacks:
+                        description: 'Number of nodes to deploy for a Cassandra deployment
+                          in each Racks. Default: 1. Optional, if not filled, used
+                          value define in CassandraClusterSpec'
+                        format: int32
+                        type: integer
+                      numTokens:
+                        description: 'NumTokens : configure the CASSANDRA_NUM_TOKENS
+                          parameter which can be different for each DD'
+                        format: int32
+                        type: integer
+                      rack:
+                        description: List of Racks defined in the Cassandra DC
+                        items:
+                          properties:
+                            labels:
+                              additionalProperties:
+                                type: string
+                              description: Labels used to target Kubernetes nodes
+                              type: object
+                            name:
+                              description: Name of the Rack
+                              type: string
+                            rollingPartition:
+                              description: The Partition to control the Statefulset
+                                Upgrade
+                              format: int32
+                              type: integer
+                            rollingRestart:
+                              description: Flag to tell the operator to trigger a
+                                rolling restart of the Rack
+                              type: boolean
+                          type: object
+                        type: array
+                    type: object
+                  type: array
+              type: object
+          required:
+          - imagepullpolicy
+          - maxPodUnavailable
+          type: object
+        status:
+          properties:
+            cassandraRackStatus:
+              additionalProperties:
+                properties:
+                  cassandraLastAction:
+                    description: 'CassandraLastAction is the set of Cassandra State
+                      & Actions: Active, Standby..'
+                    properties:
+                      Name:
+                        description: 'Type d''action a effectuer : UpdateVersion,
```

Comment:
@jsanda you're a french speaker now ? ha ha. It should be translated

Reply:
rien qu'un peu :)